### PR TITLE
[138] Product Page using HTML

### PR DESCRIPTION
**Pull request by OluwaboriShobayo**
image

image

### CODE REVIEW

**Comment by OluwaboriShobayo** [comment]
image
image

File: submissions/oluwaboriShobayo/HTML/productPage.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  <section>
+    <article>
+       <h1>Product Description</h1>
+    </article>
+    <p>
+        The Mira Dress is a multifunctional dress that can serve 
+        as a work outfit when paired with heels or flats; a casual wear
+        when paired with sandals and a church wear when paired with any
+        footwear of your choice. 
+
```
Comment (Ireti203) [inline submissions/oluwaboriShobayo/HTML/productPage.html:33]: Remove whitespace

File: submissions/oluwaboriShobayo/HTML/productPage.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+    <head>
```
Comment (Ireti203) [inline submissions/oluwaboriShobayo/HTML/productPage.html:3]: Kindly format properly.

File: submissions/oluwaboriShobayo/HTML/productPage.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        <meta charset="UTF-8">
+        <title>Product Page</title>
+    
+    </head>
```
Comment (Ireti203) [inline submissions/oluwaboriShobayo/HTML/productPage.html:7]: Here too.

File: submissions/oluwaboriShobayo/HTML/productPage.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+<body>
+    <nav>
+        <h1><strong>STYLE BY BORI</strong></h1> 
+        <a href="#">Home</a>
```
Comment (Ireti203) [inline submissions/oluwaboriShobayo/HTML/productPage.html:11]: You should have a list in your nav tag.

File: submissions/oluwaboriShobayo/HTML/productPage.html [hunk 5]
```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Product Page</title>
+
```
Comment (Ireti203) [inline submissions/oluwaboriShobayo/HTML/productPage.html:6]: Remove whitespace

File: submissions/oluwaboriShobayo/HTML/productPage.html [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    </article>
+    <p>
+        The Mira Dress is a multifunctional dress that can serve 
+        as a work outfit when paired with heels or flats; a casual wear
+        when paired with sandals and a church wear when paired with any
+        footwear of your choice. 
+
+    </p>
+  </section>
+
```
Comment (Ireti203) [inline submissions/oluwaboriShobayo/HTML/productPage.html:36]: Remove whitespace.

File: submissions/oluwaboriShobayo/HTML/productPage.html [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+        <title>Product Page</title>
+    
+    </head>
+<body>
+    <nav>
+        <h1><strong>STYLE BY BORI</strong></h1> 
+        <a href="#">Home</a>
+    </nav>
+    <hr>
+
```
Comment (Ireti203) [inline submissions/oluwaboriShobayo/HTML/productPage.html:14]: Remove whitespace.

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**